### ISSUE TITLE
Aporia 0.4.0 on Win10: Menu `Edit > Raw Preferences` error (missing `config.global.ini`)

### ISSUE THREAD

**Issue by tajmone** [nim-lang/Aporia#122]
I've installed Aporia v0.4.0 via the Nim lang installer, under Windows 10 x64.

Selecting the menu entry `Edit > Raw Preferences` produces an error in Aporia's status bar: it complains that it cannot find the file `config.global.ini` in `C:\Users\username\AppData\Roaming\Aporia`.

So I've created manually that file, and now the error is gone. Myabe the setup wizard should create that folder and file at installation time, or Aporia itself could create them when this Menu is called for and the file is not found.

I was trying to edit Aporia interface to use English lang instead of the system locale (Italian). But it looks like its imposing Italian as a locale since it sees it as the OS locale — but translation into Italian is partial, so its a mixture of English and Italian menus.

I am still unsure how this Raw Preferences file should be used; I couldn't find any reference here on how to set it. Also, it seems that changing setting from `Preferences` menu doesn't affect the `config.global.ini` file. Where are preferences being saved?

**Comment by dom96** [comment]
In that version of Aporia the config was saved when Aporia was closed. I guess you only just launched it so the config didn't get created.

The preferences file isn't automatically reloaded, you need to close and reopen it.

**Comment by tajmone** [comment]
You're right: after re-opening it the file is now populated with settings.

Has this changed in newer versions?

**Comment by dom96** [comment]
Only partially. The settings are saved more often but reloading is still not implemented.

**Comment by achesak** [comment]
That should be simple, how should reloading be implemented? Just a menu item under Raw Preferences?

**Comment by dom96** [comment]
This could be implemented as a temporary feature, in the long run Aporia should detect whether any of the files changed and ask to reload them automatically.

**Comment by tajmone** [comment]
If I understand correctly, Aporia does read the config file at startup — and if none is found, it goes for default settings. At this point it could also create the missing file, without having to wait for shutdown in this special case.

Also, when detecting that the raw-config file is being saved (within Aporia) it should reparse it so to make manual changes take effect immediately.

Maybe these 2 feature additions could fill the gap until auto-detection of file changes is implemented.

**Comment by achesak** [comment]
Try downloading and compiling the latest version. For me Aporia (master branch, Linux) creates the default settings on startup, without showing any error message or requiring a close to save.
